OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1012,7 +1017,15 @@ void Model::generateDecorations
         const SimTK::State&                         state,
         SimTK::Array_<SimTK::DecorativeGeometry>&   appendToThis) const
 {
-    Super::generateDecorations(fixed,hints,state,appendToThis);
+    ComponentList<Component> allComps = getComponentList();
+    ComponentList<Component>::const_iterator iter = allComps.begin();
+    iter++; // Skip Model itself
```

Comment:
This seems dangerous to assume. Can you verify that this first component is actually the Model and blow up if not? I'm sure it is today but it seems unlikely that anyone in the future will realize that there is code that depends on that fact. Also if it isn't there already, there should be a comment where the component list gets created that says it is critical to put the Model first. Better would be to remove that assumption from this code if possible. It seems brittle to me.